release/src/main/scripts/mass_comment.py

```diff
@@ -123,8 +123,19 @@ def getRemainingComments(accessToken, pr, initialComments):
         and 'Sickbay' not in comment[1]:
       print(comment)
       remainingComments.append(comment)
+      arm_comments = getARMTestTriggerCommands();
+      remainingComments.append(arm_comments)
   return remainingComments
 
+def getARMTestTriggerCommands():
+  '''
+  Returns all trigger commands that will start PostCommit ARM Github Actions test suites.
+  '''
+  arm_trigger_commands = []
```

Comment:
Could you also add a comment along the lines of: remove this special case once this script can handle GitHub Actions triggers

Reply:
Or should I just create a github_actions_jobs.txt now instead and move all the trigger commands there? Thanks!

Reply:
I think leaving it is fine for now (with the comment added). Solving trigger commands for actions is an open question that will need to be solved generally soon anyways

Reply:
I see, thanks! But since there are multiple versions of Java and Python, I also need to adjust the trigger command to "Run Java_Examples_Dataflow_ARM PostCommit 11" "Run Python ValidatesContainer Dataflow ARM 3.8"?

Reply:
Yeah, we should do that as well